bokehjs/test/integration/regressions.ts

```diff
@@ -1238,6 +1238,7 @@ describe("Bug", () => {
 
       const p = fig([200, 200], {y_axis_type: "log"})
       p.line(x, y, {line_width: 2})
+      p.scatter(x, y, {size: 15})
```

Comment:
Given the small size of the plot, these scatter markers overlap each other and cover the line completely. I would make them smaller and perhaps use `fill_color=None` for transparency.

Reply:
Oops missed this will try to update in the follow on PR that uses this for ranging